packages/manager/src/features/ObjectStorage/BucketDetail/AccessSelect.tsx

```diff
@@ -203,7 +207,21 @@ export const AccessSelect = React.memo((props: Props) => {
           </Link>
           .
         </Typography>
-      ) : null}
+      ) : (
+        // TODO: OBJGen2 - We need to handle link in upcoming PR
+        <Notice spacingBottom={0} spacingTop={16} variant="warning">
+          <Typography
+            sx={(theme) => ({
+              fontFamily: theme.font.bold,
+            })}
+          >
+            CORS (Cross Origin Sharing) is not available for endpoint types E2
+            and E3.
+            <br />
+            <Link to="#">Learn more</Link>
```

Comment:
```suggestion
            <Link to="#">Learn more</Link>.
```
Nit: let's add a period after learn more, outside the link text.

Question: Can we do away with the line break? `Learn more` looks a little odd all by itself when it's such a short sentence.

Reply:
I agree, let me confirm with UX